express-mutations/server.ts

```diff
@@ -63,13 +65,7 @@ app.post('/api/actors', async (req, res, next) => {
 app.put('/api/actors/:actorId', async (req, res, next) => {
   try {
     const { actorId } = req.params;
-    const { firstName, lastName } = req.query;
-    if (actorId === undefined) {
-      throw new ClientError(400, 'actorId is required');
-    }
-    if (!Number.isInteger(+actorId)) {
```

Comment:
You should still check that actorId is a number. In my previous comment I was trying to say that the check `actorId === undefined` is not necessary. The check that it's a number still is needed.

If someone uses `/api/actors/abc` then you will get `actorId` equal to `abc` and then when you use that in you SQL it will throw an exception. So check that it's a number before you use it.